sbt-scalajs-cross/src/main/scala/scalajscross/ScalaJsCross.scala

```diff
@@ -0,0 +1,41 @@
+package scalajscross
+
+import sbtcross._
+
+import sbt._
+
+import org.scalajs.sbtplugin.{ScalaJSPlugin, ScalaJSCrossVersion}
+
+case object JSPlatform extends Platform {
```

Comment:
@sjrd Are you fine with this being part of this repo or should it go to [scala-js/scala-js](https://github.com/scala-js/scala-js) ?